packages/core/js-client/src/services/Sig.ts

```diff
@@ -43,23 +43,23 @@ export const defaultSigGuard = (peerId: PeerIdB58) => {
 
 type SignReturnType =
   | {
-      error: null;
-      signature: number[];
+      error: [];
+      signature: [number[]];
       success: true;
     }
   | {
-      error: string;
-      signature: null;
+      error: [string];
+      signature: [];
```

Comment:
I am just curious why this change was required

Reply:
After we removed schema from js-client internals, there is no aqua->js casting right before service calls. So now services should work with aqua types directly.